packages/forma-36-website/src/components/Header.js

```diff
@@ -107,6 +107,16 @@ const Header = () => (
               Join the community
             </a>
           </li>
+          <li css={styles.navListItem}>
+            <a
+              css={styles.navListLink}
+              href="https://v4.f36.contentful.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
```

Comment:
```suggestion
            <a
              css={styles.navListLink}
              href="https://v4.f36.contentful.com/"
            >
```

I think we don't need to open this in a new tab
but it's just a small suggestion